dlc_developer_config.toml

```diff
@@ -28,9 +28,9 @@ benchmark_mode = false
 [build]
```

Comment:
PR description is not in the appropriate format. Can we please put it in the right format? For eg. https://github.com/aws/deep-learning-containers/pull/2372